libraries/ESPmDNS/src/ESPmDNS.cpp

```diff
@@ -82,7 +82,7 @@ void MDNSResponder::setInstanceName(String name) {
 
 void MDNSResponder::enableArduino(uint16_t port, bool auth){
     mdns_txt_item_t arduTxtData[4] = {
-        {(char*)"board"         ,(char*)ARDUINO_VARIANT},
+        {(char*)"board"         ,(char*)"esp32"},
```

Comment:
This should be reverted, the ARDUINO_VARIANT variable is injected with the board name typically. The ESPmDNS library checks explicitly for this define and if missing defines it itself as you have done here: https://github.com/espressif/arduino-esp32/blob/master/libraries/ESPmDNS/src/ESPmDNS.h#L48-L51

Reply:
yeah, why would you change that?

Reply:
I don't know why all these commits are part of this pull request,  these are from months ago,  and already reverted locally.
It also looks like that code isn't there anymore